docs/book/features/volume_expansion.md

```diff
@@ -127,36 +85,30 @@ spec:
   volumes:
   - name: test-volume
     persistentVolumeClaim:
-      claimName: example-vanilla-block-pvc
+      claimName: example-block-pvc
 ```
 
-Kubelet will trigger the volume expansion of filesystem when the PVC is used by the Pod.
-
 ```bash
 kubectl create -f example-pod.yaml
-pod/example-vanilla-block-pod created
+pod/example-block-pod created
 ```
 
-```bash
-kubectl get pod
-NAME                        READY STATUS RESTARTS   AGE
-example-vanilla-block-pod   0/1 ContainerCreating 0          37s
+The Kubelet on the node will trigger the volume expansion of filesystem when the PVC is used by the Pod.
```

Comment:
trigger the volume expansion of filesystem -> trigger the filesystem expansion on the volume
when the PVC is used by the Pod ->  when the PVC is attached to the Pod

Reply:
Done